apps/system/js/edge_swipe_detector.js

```diff
@@ -104,18 +104,18 @@
           // XXX: Use this.appWindowManager instead if
           // we become part of appWindowManager submodules.
           // i.e., Service.query('getTopMostUI') === this.parent
-          if (Service.query('getTopMostUI') &&
-              Service.query('getTopMostUI').name === 'AppWindowManager') {
-            if (Service.query('getTopMostWindow') &&
-                !Service.query('getTopMostWindow').isHomescreen &&
+          var topMostUI = Service.query('getTopMostUI');
+          var topMostWindow = Service.query('getTopMostWindow');
+          if (topMostUI && topMostUI.name === 'AppWindowManager') {
+            if (topMostWindow && !topMostWindow.isHomescreen &&
```

Comment:
Do we need to check both topMostWindow and topBottomWindow isHomescreen? I think with the topBottom should be enough, isn't it?